apps/sms/test/marionette/conversation_test.js

```diff
@@ -16,10 +17,20 @@ marionette('Conversation Panel Tests', function() {
     '/mocks/mock_navigator_moz_mobile_message.js',
     '/mocks/mock_navigator_moz_contacts.js'
   ];
+  var apps = {};
+  apps[MessagesActivityCaller.ORIGIN] = __dirname + '/apps/activitycaller';
 
-  var client = marionette.client();
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'focusmanager.testmode': true
```

Comment:
I'm wondering if we still need this, I remember when I was creating one of my first tests I've copied it from calendar tests. There was a comment like ```Do not require the B2G-desktop app window to have focus (as per the system window manager) in order to do focus-related things.```

Hey @JohanLorenzo do you know if\when we actually need it?